pkgs/development/python-modules/pyscf/default.nix

```diff
@@ -1,6 +1,7 @@
 {
   buildPythonPackage,
   lib,
+  stdenv,
```

Comment:
Are you using `stdenv` anywhere in this derivation?

Reply:
No, removed.